src/api/index.ts

```diff
@@ -167,25 +173,46 @@ router.get('/:installationId',
     } catch (err) {
       req.log.error(err);
       res.status(500).json(err);
+    } finally {
+      const elapsed = Date.now() - requestStart;
+      const tags = {
+        path:'/:installationId',
+        method: 'GET',
+        status: res.status.toString(),
+      };
+
+      statsd.histogram('get.installationId', elapsed, tags);
     }
   });
 
 router.get('/:installationId/repoSyncState.json',
   check('installationId').isInt(),
   returnOnValidationError,
   async (req: Request, res: Response): Promise<void> => {
+    const requestStart = Date.now();
```

Comment:
We need to add it as a middleware function to avoid code duplication.
As it is done, for example, in: src/config/enhance-octokit.ts:33

Reply:
Would be also great to unify it across endpoints, so we'll have a common code which emits app response time metrics